lib/pbench/server/indexing_tarballs.py

```diff
@@ -218,13 +223,22 @@ def process_tb(self, tarballs):
                 def sigquit_handler(*args):
                     sigquit_interrupt[0] = True
 
+                sighup_interrupt = [False]
+
+                def sighup_handler(*args):
+                    sighup_interrupt[0] = True
+
                 signal.signal(signal.SIGQUIT, sigquit_handler)
+                signal.signal(signal.SIGHUP, sighup_handler)
+                idx = 0
```

Comment:
It occurs to me that `idx` isn't a particularly meaningful name anymore. It's a "count of processed tarballs". But while it might be nice to make this more clear should you make some other change, I don't want to push you to make another commit for this small detail.